lib/platform/bitbucket/index.spec.ts

```diff
@@ -803,6 +803,71 @@ describe('platform/bitbucket/index', () => {
       await bitbucket.updatePr({ number: 5, prTitle: 'title', prBody: 'body' });
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
+    it('removes reviewers no longer member of the workspace when updating pr', async () => {
```

Comment:
test need assertion!

Reply:
Ah it was the `expect(httpMock.getTrace()).toMatchSnapshot();` 😅
Should I just add it back? What would be the alternative approach?

Reply:
I have added `expect(httpMock.getTrace()).toMatchSnapshot();` back for now